architecture/evm/evm_state_poller.go

```diff
@@ -75,8 +75,9 @@ func NewEvmStatePoller(
 ) *EvmStatePoller {
 	lg := logger.With().Str("upstreamId", up.Config().Id).Str("component", "evmStatePoller").Logger()
 
-	lbs := sharedState.GetCounterInt64(fmt.Sprintf("latestBlock/%s", common.UniqueUpstreamKey(up)))
-	fbs := sharedState.GetCounterInt64(fmt.Sprintf("finalizedBlock/%s", common.UniqueUpstreamKey(up)))
+	// TODO: replace with integrity config
+	lbs := sharedState.GetCounterInt64(fmt.Sprintf("latestBlock/%s", common.UniqueUpstreamKey(up)), 1024)
```

Comment:
For now maybe let's put this in a const?
```suggestion
	lbs := sharedState.GetCounterInt64(fmt.Sprintf("latestBlock/%s", common.UniqueUpstreamKey(up)), DefaultToleratedBlockHeadDownDrift)
```

Also for the value (1024), do we need to be more strict or it's better to leave it as is. This is mainly from the same upstream (which means maximum 'normal' down drift we should expect is the evm poller interval / fastest block production on fastest chain)

Using too low of value can create false-positives hence creating flip-flop values for latest block, and using too high of value can disregard small re-orgs (e.g. if a chain is reorged in past 8-16 blocks)